src/lib/media-query/media-observable.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {map} from 'rxjs/operators/map';
+import {filter} from 'rxjs/operators/filter';
+
+import {MediaChange} from './media-change';
+import {BreakPointRegistry} from './breakpoints/break-point-registry';
+import {MatchMedia} from './match-media';
+import {BreakPoint} from './breakpoints/break-point';
+import {mergeAlias} from '../utils/add-alias';
+
+@Injectable()
+export class MediaObservable {
+
+  /**
+   * Should we announce gt-<xxx> breakpoint activations ?
+   */
+  filterOverlaps = true;
+
+  constructor(private breakpoints: BreakPointRegistry,
+              private matchMedia: MatchMedia) {
+    this._registerBreakPoints();
+  }
+
+  /**
+   * Test if specified query/alias is active.
+   */
+  isActive(alias): boolean {
+    let query = this._toMediaQuery(alias);
+    return this.matchMedia.isActive(query);
+  }
+
+  observe(): Observable<MediaChange> {
```

Comment:
The name could be more specific, and you might consider whether or not you want this to be a property. Since Observables do nothing until you execute them, they are more like functions than Promises are. So a method that returns an Observable is more like a higher order function, where a property that returns an Observable is more like a method. If that makes sense.

I'm not saying that this fits this use case, it's just something I see.

Reply:
For the case of this service, I think it makes sense, but I'll talk it over with @ThomasBurleson to see if we will ever need parameters for this like with `MatchMedia`